readthedocs/proxito/views/mixins.py

```diff
@@ -186,22 +186,21 @@ def _spam_response(self, request, project):
 
 
 class ServeRedirectMixin:
-
-    def system_redirect(self, request, final_project, lang_slug, version_slug, filename):
+    def system_redirect(
+        self, request, final_project, version_slug, filename, external=False
```

Comment:
It would be good to document what `external` means here. Does this mean it's a URL from a PR or it's a custom domain? 🤔 Depending on your answer, it may makes sense to use a more descriptive name for the variable as well.

Reply:
the first, what about `is_external_version` or `external_version`?

Reply:
Either is fine. Both are a lot more explicit than just `external` 👍🏼